lib/rucio/cli/bin_legacy/rucio.py

```diff
@@ -1467,22 +1466,26 @@ def list_rules(args, client, logger, console, spinner):
             rules.append(client.get_replication_rule(rule_id))
     elif args.did:
         scope, name = get_scope(args.did, client)
-        meta = client.get_metadata(scope=scope, name=name)
-        rules = client.list_did_rules(scope=scope, name=name)
+        if not name.endswith('*'):
```

Comment:
This assumes wildcards only appear at the end. What if someone queries `file_*_test` or `*_important`? We might want:

```suggestion
     if "*" not in name:  # No wildcards at all
```